extensions/amp-animation/0.1/test/test-amp-animation.js

```diff
@@ -33,9 +33,13 @@ describes.sandboxed('AmpAnimation', {}, (env) => {
         ioCallback = callback;
         ioCallbacks.push(callback);
       }
-      observe() {
+      observe(target) {
         ioCallback([
-          {isIntersecting: (config && config.isIntersecting) ?? true},
```

Comment:
Does `??` get transformed properly? I'm not sure which new syntax we can / can't use yet.

Also, more important: why does `test-amp-animation` need any changes at all?

Reply:
Yes. The `??` is fully available for us now. As for the `test-amp-animation` changes - it's tricky. This test has stubbed out the full implementation of the `IntersectionObserver`, but it was passing really weird data.